sql/parser/expr.go

```diff
@@ -233,17 +231,13 @@ func (node *CoalesceExpr) String() string {
 }
 
 // IntVal represents an integer.
-type IntVal int64
+type IntVal struct {
+	Val int64
```

Comment:
I started out with these fields private, but we construct `IntVals` outside of the parser package.